src/components/folio-number-input/FolioNumberInput.stories.ts

```diff
@@ -1,18 +1,21 @@
-import type { Meta } from '@storybook/vue'
+import type { Meta } from '@storybook/vue3'
 import { FolioNumberInput } from './index'
 
 const meta: Meta<typeof FolioNumberInput> = {
   title: 'component/FolioNumberInput'
 }
 export default meta
 
-const Template = (args, { argTypes }) => ({
-  props: Object.keys(argTypes),
+const Template = (args) => ({
   components: { FolioNumberInput },
-  template: '<folio-number-input v-bind="$props" />' // $props comes from args below
+  setup () {
+    return { args }
+  },
+  template: '<folio-number-input v-bind="args" />'
 })
 
 export const Default = Template.bind({})
 Default['args'] = {
-  folioNumber: '01234567890123456789012345678901234567890123456789ABC@123/*!$ ...'
+  folioNumber: '01234567890123456789012345678901234567890123456789ABC@123/*!$ ...',
+  validate: false
```

Comment:
Added to show the storybook toggle for easier testing. Same for in the StaffPayment storybook file.